src/main/java/be/coronalert/statistics/vaccinations/VaccinationsPoller.java

```diff
@@ -0,0 +1,48 @@
+package be.coronalert.statistics.vaccinations;
+
+import be.coronalert.statistics.config.StatisticsServiceConfig;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.Map;
+
+import static java.util.stream.Collectors.groupingBy;
+import static java.util.stream.Collectors.summingInt;
+
+@Component
+public class VaccinationsPoller {
+  @Autowired
+  private StatisticsServiceConfig statisticsServiceConfig;
+
+  @Autowired
+  private ObjectMapper objectMapper;
+
+  /**
+   * Retrieves the sciensano vaccinations data.
+   *
+   * @return A map containing the amount of vaccinations for dose 1 and 2.
+   */
+  public Map<VaccinationLevel, Integer> pollResults() throws IOException {
+    final String PARTIALLY_VACCINATED_WITH_ONE_DOSE = "A";
```

Comment:
Not the correct place to create these constants.

I would have placed them on Vaccination.
But it instead of defining constants it would be better to just map the JSON dose to enums (see https://stackoverflow.com/questions/31689107/deserializing-an-enum-with-jackson). Think you can also specify a `JsonEnumDefaultValue` if these is an "unkown" value in the JSON that cannot be mapped.
That way we don't need constants (we can use the enums)